requirements/constraints.txt

```diff
@@ -15,4 +15,4 @@ pluggy<=0.13.1          # diff-cover, tox, and pytest are creating version confl
 
 # pinning celery to latest release
 celery<6.0
-click<8.0.0
```

Comment:
nit: could we just remove this line? It seems like celery constrains this themselves, also: https://github.com/celery/celery/blob/83747fdbe8a751713f702bf765fef31d08229dd9/requirements/default.txt

Reply:
good idea